### PR TITLE
Add an ActorType dropdown

### PR DESCRIPTION
**Pull request by clayplumridge**
Add a rhythm mixin

image
image

### CODE REVIEW

File: src/styles.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Theme } from "@material-ui/core";
+
+export const Mixins = {
+    rhythm: (theme: Theme, spacing: number, direction: "row" | "column") => {
```
Comment (clayplumridge) [inline src/styles.ts:8]: Need to add a doc comment here

File: src/styles.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        const selector = "& > *:not(:first-child)";
```
Comment (alex-power) [inline src/styles.ts:10]: dafaq

File: src/render/views/CreateActorTemplateView.tsx [hunk 3]
```diff
@@ -69,6 +81,19 @@ export const CreateActorTemplateView: React.FC<{}> = () => {
                 value={initiativeModifier}
             />
 
+            <Select
+                label="Actor Type"
+                onChange={ev => setActorType(ev.target.value as ActorType)}
```
Comment (alex-power) [inline src/render/views/CreateActorTemplateView.tsx:77]: What's `ev`?

File: src/render/views/CreateActorTemplateView.tsx [hunk 4]
```diff
@@ -28,6 +31,15 @@ const useStyles = makeStyles(theme =>
     })
 );
 
+function toDisplayName(actorType: ActorType) {
```
Comment (alex-power) [inline src/render/views/CreateActorTemplateView.tsx]: this seems unnecessary

File: src/render/views/CreateActorTemplateView.tsx (hunk 3 above)
Comment (clayplumridge) [reply src/render/views/CreateActorTemplateView.tsx:77]: The change event

**Pull request merged**